### ISSUE TITLE
Two albums showing contents of a single folder

### ISSUE THREAD

**Issue by paramjit** [kollerlukas/Camera-Roll-Android-App#124]
Hi,

First off, great work! Thanks for your time, this app really has good potential.

I have checked the previous issues. I think I found a bug: I have a folder say "oldies" which has 16 photos , in the sd card. Camera roll shows two different albums for it with the same name, showing 10 and 6 photos respectively, exclusive of each other. I think the expected behaviour is to show one album per folder. I have not created any virtual albums. The same happens for two more folders, which are apparently split among two albums. I already tried refreshing, clearing data, restarting, etc.

Some details.

- OS: LineageOS 14.1
- Device: Redmi Note 4

**Comment by paramjit** [comment]
I might add that this problem seems to be solved by using the app's MediaRetriever. But as soon as I stop the MediaRetriever the multiple albums for a single folder reappear.

**Comment by kollerlukas** [comment]
Can you open the images from both folders?

**Comment by paramjit** [comment]
Yes, I was able to open images from both folders. Just to reiterate, the single folder in sd card seemed to be split up into two disjoint albums in the app.
